drivers/modem/modem_context.h

```diff
@@ -19,15 +20,16 @@
 #include <net/net_ip.h>
 #include <sys/ring_buffer.h>
 #include <drivers/gpio.h>
+#include "modem_sms.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 #define MODEM_PIN(name_, pin_, flags_) { \
-	.dev_name = name_, \
-	.pin = pin_, \
-	.init_flags = flags_ \
+		.dev_name = name_,	 \
+		.pin = pin_,		 \
+		.init_flags = flags_	 \
```

Comment:
The whitespace changes here and below are fine but should be placed in separate commit as they are not really related to SMS changes.